spec/services/doc_auth/mock/doc_auth_mock_client_spec.rb

```diff
@@ -174,4 +175,50 @@
 
     expect(post_images_response).to be_a(DocAuth::Response)
   end
+
+  describe 'generate response for failure indicating http status' do
+    it 'generate network error response for status 500 when post image' do
```

Comment:
When check out this branch and try the `yml` files as described in the PR testing plan, I don't see the errors I'd expect. Instead of the `technical difficulties` message, I see:

440: `Your image size is too large or too small. Please add images of your ID that are about 2025 x 1275 pixels.`
438: `The image file that you added is not supported. Please take new photos of your ID and try again.`
500: `Try taking new pictures.`

Maybe I'm misunderstanding and this is expected, but I thought I'd see the `network_error` of `We are having technical difficulties on our end. Please try to submit your images again later.`

Reply:
Hmm looking again at your test though, maybe I'm wrong and this is expected, as in line `221` of this file.